hpx/include/traits.hpp

```diff
@@ -54,13 +54,13 @@
 #include <hpx/traits/is_component.hpp>
 #include <hpx/traits/is_continuation.hpp>
 #include <hpx/traits/is_distribution_policy.hpp>
-#include <hpx/traits/is_executor.hpp>
-#include <hpx/traits/is_executor_parameters.hpp>
-#include <hpx/traits/is_future.hpp>
+#include <hpx/is_instance/traits/is_executor.hpp>
+#include <hpx/is_instance/traits/is_executor_parameters.hpp>
+#include <hpx/is_instance/traits/is_future.hpp>
 #include <hpx/traits/is_future_range.hpp>
 #include <hpx/traits/is_future_tuple.hpp>
-#include <hpx/traits/is_launch_policy.hpp>
-#include <hpx/traits/is_timed_executor.hpp>
+#include <hpx/is_instance/traits/is_launch_policy.hpp>
+#include <hpx/is_instance/traits/is_timed_executor.hpp>
```

Comment:
I think all five of these traits should stay where they are. They belong with the executor/future (`execution`)  module.

Reply:
Yep I revert these